zh/deploy-tidb-cluster-across-multiple-kubernetes.md

```diff
@@ -0,0 +1,389 @@
+---
+title: 跨多个 Kubernetes 集群部署 TiDB 集群
+summary: 本文档介绍如何实现跨多个 Kubernetes 集群部署 TiDB 集群
+---
+
+# 跨多个 Kubernetes 集群部署 TiDB 集群
+
+跨多个 Kubernetes 集群部署 TiDB 集群，是指在多个网络互通的 Kubernetes 集群上部署**一个** TiDB 集群，集群各个组件分布在多个 Kubernetes 集群上，实现在 Kubernetes 集群间容灾。所谓 Kubernetes 集群网络互通，是指 Pod IP 在任意集群内和集群间可以被互相访问，Pod FQDN 记录在集群内和集群间均可被解析。
+
+## 前置条件
+
+您需要配置 Kubernetes 的网络插件和 DNS 插件，使得 Kubernetes 集群满足以下条件：
+
+- 各 Kubernetes 集群上的 TiDB 组件有能力访问集群内和集群间所有 TiDB 组件的 Pod IP 
+- 各 Kubernetes 集群上的 TiDB 组件有能力解析集群内和集群间所有 TiDB 组件的 Pod FQDN
+
+## 目前支持场景
+
+- 新部署跨多个 Kubernetes 集群的 TiDB 集群
+- 在其他 Kubernetes 集群上部署开启此功能的新集群加入同样开启此功能的集群
+
+## 实验性支持场景
+
+- 对已有数据的集群从未开启此功能状态变为开启此功能状态，生产使用建议通过数据迁移完成此需求
+
+## 不支持场景
+
+- 两个已有数据集群互相连通，对于这一场景应通过数据迁移完成
+
+## 跨多个 Kubernetes 集群部署 TiDB 集群
+
+部署跨多个 Kubernetes 集群的 TiDB 集群，默认您已部署好此场景所需要的 Kubernetes 集群，在此基础上进行下面的部署工作。
+
+下面以部署两个集群为例进行介绍，其中集群 1 为初始集群，按照下面给出的配置进行创建，集群 1 正常运行后，按照下面给出配置创建集群 2，等集群完成创建和部署工作后，两个集群正常运行。
+
+### 部署初始集群
+
+通过如下命令部署初始化集群，实际使用中需要根据您的实际情况设置 `cluster1_name` 和 `cluster1_domain` 变量的内容，其中 `cluster1_name` 为集群 1 的集群名称，`cluster1_domain` 为集群 1 的 [Cluster Domain](https://kubernetes.io/docs/tasks/administer-cluster/dns-custom-nameservers/#introduction), `cluster1_namespace` 为集群 1 的命名空间。
+
+```bash
```

Comment:
Can you separate commands from the config? This looks confusing.

Reply:
I think it is one-step deployment command, not multi-step commands in one code block. Meanwhile, if the user runs these commands separately, and the terminal session has gone, the variables in the deployment may be mistaken.

Reply:
fixed